client/apps/edit/public/icons/edit_image_set_slideshow.svg

```diff
@@ -1 +1 @@
-<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 44 44"><defs><style>.cls-1{fill:#231f20;}</style></defs><title>Untitled-1</title><path class="cls-1" d="M6760.67,551.72h-32v-32h32v32Zm-30-2h28v-28h-28v28Z" transform="translate(-6728.67 -507.72)"/><polygon class="cls-1" points="38 37 36 37 36 8 7 8 7 6 38 6 38 37"/><polygon class="cls-1" points="44 31 42 31 42 2 13 2 13 0 44 0 44 31"/></svg>
\ No newline at end of file
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 44 44"><defs><style>.cls-1{fill:#ffffff;}</style></defs><title>Untitled-1</title><path class="cls-1" d="M6760.67,551.72h-32v-32h32v32Zm-30-2h28v-28h-28v28Z" transform="translate(-6728.67 -507.72)"/><polygon class="cls-1" points="38 37 36 37 36 8 7 8 7 6 38 6 38 37"/><polygon class="cls-1" points="44 31 42 31 42 2 13 2 13 0 44 0 44 31"/></svg>
```

Comment:
I've mentioned this before but in times like this, feel free to change the classname within the SVG if you need to access the fill or other properties. That way you should be able to remove the embedded style tag entirely and use the classname for setting properties via Styl instead of inline. Not necessary here, but just something to think about.